model/ValueObjects/TaskLogCategorizedStatus.php

```diff
@@ -78,6 +79,7 @@ public static function createFromString($status)
                 return TaskLogCategorizedStatus::inProgress();
                 break;
 
+            case TaskLogInterface::STATUS_ARCHIVED:
```

Comment:
Having archived and completed as the same category can lead to some confusion. An archived record is kind of a "deleted" record, not visible for the user. So I would suggest, just simply create a new separated category for STATUS_ARCHIVED like for the other ones. For example TaskLogCategorizedStatus::archieved(); we don't  have this because we didn't needed so far.